solr/core/build.gradle

```diff
@@ -115,9 +115,6 @@ dependencies {
 
   testImplementation 'org.slf4j:jcl-over-slf4j'
 
-  // JWT Auth plugin
-  api 'org.bitbucket.b_c:jose4j'
```

Comment:
The jose lib was defined as `api` in solr-core. When I moved it into the contrib (as implementation), suddenly all contribs complained they could not find `slf4j`. I considered changing `org.slf4j:slf4j-api` to `api` scope in solr-core, to give all contribs depending on solr-core access to the logger. But then I instead added `slf4j-api` to all of them explicitly. Any preference?

Reply:
My opinion is that if Solr core is using slf4j  for logging and it's something encouraged for contribs then I think it's fine to put it in the api configuration (as a transitive, exported dependency).

Reply:
+1 to what Dawid says -- definitely an "api" dependency.  Feel free to ping me Jan on dependency matters; I'm glad Dawid is still here to help.

Reply:
Done.

Reply:
> I'm glad Dawid is still here to help.

Yeah, kind of. I'm a bit terrified at the size of unread messages in dev/solr when I wake up each morning. Seems like 800 guys are working on it full time! Or just Jan, but has a 800 hours day on the planet where he lives... :)

Also, more seriously and interestingly, I don't get e-mails about review requests from github... I'll check the settings but if you have something important, use a mention (these I do get) or just call out via e-mail explicitly. Odd.